build.gradle

```diff
@@ -179,6 +179,7 @@ project(":samza-core_$scalaVersion") {
     compile("com.101tec:zkclient:$zkClientVersion") {
       exclude module: 'junit:junit'
     }
+    compile "org.apache.avro:avro:$avroVersion"
```

Comment:
We don't want an avro dependency in samza-core.

Reply:
removed & code refactored as discussed offline